arm-web/2016-06-01/swagger/connectionGateways.json

```diff
@@ -372,6 +425,11 @@
           "type": "string",
           "readOnly": true,
           "description": "Gets the resource type."
+        },
```

Comment:
what's the intent of "ResourceReference" it appears to have everything that "Resource" has except for tags?